pages/common/terraform-plan.md

```diff
@@ -0,0 +1,32 @@
+# terraform plan
+
+> Generate and show an execution plan.
+> More information: <https://www.terraform.io/docs/cli/commands/plan.html>.
+
+- Generate and show an execution plan:
+
+`terraform plan`
+
+- Destroy mode - Show plan to destroy all remote objects that currently exist:
```

Comment:
I don't exactly understand what this does, but is this still correct?
```suggestion
- Show a plan to destroy all remote objects that currently exist:
```

Reply:
Yes, it's almost right.